src/EditorFeatures/CSharp/Formatting/Indentation/CSharpIndentationService.Indenter.cs

```diff
@@ -251,6 +251,17 @@ private IndentationResult GetIndentationBasedOnToken(SyntaxToken token)
                             return GetIndentationFromCommaSeparatedList(token);
                         }
 
+                    case SyntaxKind.CloseParenToken:
+                        {
+                            if (token.Parent.IsKind(SyntaxKind.ArgumentList))
+                            {
+                                return GetDefaultIndentationFromToken(token.Parent.GetFirstToken(includeZeroWidth: true));
+                            }
+
+                            // default case
+                            return GetDefaultIndentationFromToken(token);
```

Comment:
if this is the default case, why not `goto case default;`?

Reply:
What do you mean exactly? Just remove the `// default case`, and `return ...` lines, so it breaks into the `default` case?

Reply:
@tiesmaster You can't implicitly fall through, but you can replace the comment and duplicated call with a `goto default` statement like this one:

https://github.com/dotnet/roslyn/blob/4239268b20a47c91573cdadfc9b4a4d359a25860/src/Workspaces/Core/Portable/Utilities/AbstractSpeculationAnalyzer.cs#L349

Reply:
@sharwell I've updated the PR. Please have another look.

Reply:
As you can see, I only updated the places where it's the end of case, but there are a couple more places but they're not at the end of the case, inside branches ([L190](https://github.com/dotnet/roslyn/pull/33397/files#diff-439e28ddaa46ef9cc35459839703220aR190), and [L203](https://github.com/dotnet/roslyn/pull/33397/files#diff-439e28ddaa46ef9cc35459839703220aR203) in the PR). Would you also replace those?